pkg/webhook/admission/defaulter.go

```diff
@@ -27,7 +27,7 @@ import (
 // Defaulter defines functions for setting defaults on resources
 type Defaulter interface {
 	runtime.Object
-	Default()
+	Default() error
```

Comment:
/hold

Defaulting shouldn't error, use `admission.Validator` for that.

Reply:
my commits are overwritten by the pullbot.
what I want is similar to the Validator interface, and multiplex the same path.
can I add this?
```
type Mutator interface {
	runtime.Object
	MutatorCreate() error
	MutatorUpdate(old runtime.Object) error
}
```